src/main.rs

```diff
@@ -3,9 +3,34 @@ use chrono::{Duration, Utc};
 use clap::{AppSettings, Clap};
 use fehler::throws;
 use octocrab::models::Repository;
+use serde_derive::Deserialize;
+use std::path::Path;
 mod token;
 mod util;
 
+#[derive(Debug, Deserialize, Default)]
+struct Config {
+    github: GithubConfig,
+}
+
+#[derive(Debug, Deserialize, Default)]
+struct GithubConfig {
+    org: String,
```

Comment:
This should (I think) be `Option<String>`

Reply:
yeah, It's clearer